src/views/Events/index.js

```diff
@@ -0,0 +1,202 @@
+              Filters
+            </Button>
+          </Grid>
+          <Grid item xs={11} md={12} lg={5}>
+            <Collapse in={this.state.open} timeout="auto" unmountOnExit>
+              <FormGroup row>
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.academics}
+                      onChange={this.filterEvents('academics')}
+                    />
+                }
+                  label="Academics"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.admissions}
+                      onChange={this.filterEvents('admissions')}
+                    />
+                 }
+                  label="Admissions"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.art}
+                      onChange={this.filterEvents('art')}
+                    />
+                 }
+                  label="Arts"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.calendar}
+                      onChange={this.filterEvents('calendar')}
+                    />
+                 }
+                  label="Calendar Events"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.cec}
+                      onChange={this.filterEvents('cec')}
+                    />
+                 }
+                  label="CEC"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.chapelOffice}
+                      onChange={this.filterEvents('chapelOffice')}
+                    />
+                 }
+                  label="Chapel Office"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.studentLife}
+                      onChange={this.filterEvents('studentLife')}
+                    />
+                 }
+                  label="Student Life"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.fair}
+                      onChange={this.filterEvents('fair')}
+                    />
+                 }
+                  label="Fair or Expos"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.sports}
+                      onChange={this.filterEvents('Sports')}
+                    />
```

Comment:
The `Athletics` filter doesn't seem to be working. The checkbox isn't changing and the events aren't being filtered.
Also, it is somewhat confusing to have `sports` when the actual filter label is called `Athletics`.